pkg/api/server.go

```diff
@@ -41,7 +37,7 @@ func NewServer(ctx context.Context, config ServerConfig) *http.Server {
 
 	database := Database{
 		Distro: newBucket(ctx, config.DistroBucketName),
-		Parts:  parts.NewParts(config.RedisNamespace + ":parts"),
+		Parts:  parts.NewParts(config.RedisNamespace),
```

Comment:
The methods on Parts also prepend ":parts", so in redis it became `vvgo:parts:parts:id` lol